src/routes/docs/products/ai/+layout.svelte

```diff
@@ -99,6 +99,10 @@
                 {
                     label: 'LangChain',
                     href: '/docs/products/ai/integrations/langchain'
+                },
+                {
+                    label: 'fal.ai',
```

Comment:
Maybe the label should use Title case as well. It looks weird when rendered

<img width="227" alt="Screenshot 2024-04-25 at 3 04 05 PM" src="https://github.com/appwrite/website/assets/20852629/93af9a76-e9ab-4b7c-99d9-7bcefdcc1ab3">

Reply:
Going to stick with lowercase `fal.ai` as this is what they use